backend/api/messaging/index.js

```diff
@@ -13,7 +13,7 @@ const router = express.Router({ mergeParams: true });
 router.post(
   '/message',
   asyncHandler(async (req, res) => {
-    const { message, sender, channel, image } = req.body;
+    const { message, sender, channel, image, id } = req.body;
```

Comment:
Do not pass `id` from client (Bad practice because anyone can change id from the client-side).